src/MessagePack.UnityClient/Assets/Scripts/MessagePack/SequenceReaderExtensions.cs

```diff
@@ -41,6 +41,50 @@ internal static unsafe bool TryRead<T>(ref this SequenceReader<byte> reader, out
             return true;
         }
 
+#if UNITY_ANDROID
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static unsafe bool TryRead(ref this SequenceReader<byte> reader, out long value)
```

Comment:
Can you add an xml doc comment on this method explaining why the method is here so we can see that looking at code without digging through github issues or git history?

Reply:
okay, I've added.